CityOfDetroit/bloom
# feat: automatically set region based on neighborhood

## Issue

- Closes #1294 

## Description

Automatically selects the mapped region when a neighborhood value is selected in the listing form.

Also moved two files that had to be touched `filters` and `region` from ui-components into shared-helpers due to backend dependencies.

## How Can This Be Tested/Reviewed?

In the listing form, change the neighborhood value and the region value should populate with its mapped value. Ensure you can change it manually from its mapped value, save it, and that it re-populates with the custom value.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
